moped-editor/src/views/projects/projectsListView/ProjectsListViewMap.js

```diff
@@ -1,11 +1,98 @@
-import React from "react";
+import React, { useEffect } from "react";
+import { useQuery } from "@apollo/client";
 import ProjectsMap from "./components/ProjectsMap";
+import Alert from "@mui/material/Alert";
+import { GET_PROJECTS_GEOGRAPHIES } from "src/queries/project";
+import { styleMapping } from "../projectView/ProjectStatusBadge";
 
-const ProjectsListViewMap = () => {
+const ProjectsListViewMap = ({
+  mapQuery,
+  fetchPolicy,
+  setIsMapDataLoading,
+}) => {
   /* Store map instance to call Mapbox GL methods where needed */
   const mapRef = React.useRef();
 
-  return <ProjectsMap ref={mapRef} />;
+  const { data: projectMapViewData } = useQuery(mapQuery, {
+    fetchPolicy,
+  });
+
+  /* Build object to get project data needed for projectsFeatureCollection by project id  */
+  const projectDataById = React.useMemo(
+    () =>
+      projectMapViewData
+        ? projectMapViewData?.project_list_view.reduce((acc, project) => {
+            return {
+              ...acc,
+              [project.project_id]: project,
+            };
+          }, {})
+        : {},
+    [projectMapViewData]
+  );
+
+  /* Build array of project ids to request project geography */
+  const projectIds = React.useMemo(
+    () => Object.keys(projectDataById),
+    [projectDataById]
+  );
+
+  const {
+    loading,
+    error,
+    data: projectsGeographies,
+  } = useQuery(GET_PROJECTS_GEOGRAPHIES, { variables: { projectIds } });
+
+  /* Update state tied to loading spinner in SearchBar component */
+  useEffect(() => {
+    setIsMapDataLoading(loading);
+  }, [loading, setIsMapDataLoading]);
+
+  /* Build feature collection to pass to the map, add project_geography attributes to feature properties along with status color */
+  const projectsFeatureCollection = React.useMemo(() => {
+    const projectGeographiesFeatureCollection =
+      projectsGeographies?.project_geography
+        ? projectsGeographies?.project_geography.reduce(
+            (acc, projectGeography) => {
+              const phaseKey =
+                projectDataById[projectGeography.project_id]?.current_phase_key;
```

Comment:
i wonder if/how we should incorporate the component phase "override" into the map. e.g., if one component of a project is complete and the rest of the project is in construction, should the complete component be styled differently? would that be confusing if the list has been filtered on project phase = "construction"? etc.. 🤔

Reply:
awesome, yes, glad that we sync'd on this earlier too. I'm going to create an enhancement issue out of this since it came to mind for both you and Nathan to get a better feel for if seeing different component phases would be confusing or not. or how to make it more obvious that the map is displaying the overrides. maybe a switch that enables that would be an option to explore too.

Reply:
Now captured in https://github.com/cityofaustin/atd-data-tech/issues/17638